lib/tasks/migrate_homepage.rake

```diff
@@ -0,0 +1,38 @@
+require 'gds_api/publishing_api'
+require 'highline'
+
+desc "Migrate the homepage"
+task migrate_homepage: :environment do
+  cli = HighLine.new
+  cli.say cli.color(
+    "Publishing the homepage will make the old service manual inaccessible. Continue?",
+    :red
+  )
+  exit unless cli.agree "Are you sure you wish to continue?"
+
+  publishing_api_v1 = GdsApi::PublishingApi.new(
```

Comment:
We're using v1 because of something to do with the path reservation yes? Could you remind me quickly.. I has forgot.